File: Selenium/Project/activity2.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.junit.jupiter.api.Assertions;


// Get the url of the header image

public class activity2 {
    public static void main(String[] args) throws InterruptedException {
        // Install driver
        WebDriverManager.firefoxdriver().setup();

        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "NULL");
        WebDriver driver = new FirefoxDriver();

        driver.get("http://alchemy.hguy.co/crm");
        String header = driver.findElement(By.xpath("//div[@class='companylogo']/img")).getAttribute("src");
        System.out.println(header);

        driver.quit();
    }
}

File: Selenium/Project/activity2.py
# Get the url of the header image

from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Firefox()
driver.get("http://alchemy.hguy.co/crm")

header = driver.find_element(By.XPATH, "//div[@class='companylogo']/img").get_attribute("src")
print(header)
driver.quit()